aten/src/ATen/Dispatch.h

```diff
@@ -155,6 +163,24 @@ inline void deprecated_AT_DISPATCH_ALL_TYPES_AND_HALF_AND_COMPLEX() {}
     }                                                                                                     \
   }()
 
+// This macro should be used to dispatch bfloat16 on ROCm.
+// On non-ROCm platforms, it ignores bfloat16 data type.
+#define AT_DISPATCH_FLOATING_TYPES_AND_BFLOAT16_AND(SCALARTYPE, TYPE, NAME, ...)                                       \
```

Comment:
You should update this one too right?

Reply:
Does changing that to `AT_DISPATCH_FLOATING_TYPES_AND_ROCM_BFLOAT16_AND` look ok?